components/project-controller/development/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
+- https://github.com/konflux-ci/project-controller/config/default?ref=edfd123a157000714b19591421e2f93eed70831d
+
+images:
+- name: quay.io/redhat-appstudio/project-controller
```

Comment:
@hmariset Lets remove the prefix here so the original name is `project-controller` and set that name as the image name in the upstream repo in konflux-ci/project-controller#13.

That way, we don't have a Red Hat specific name in the upstream repo, but the name is not so generic that it will collide with other controllers.

Reply:
@ifireball Should I rename the image to quay.io/project-controller or quay.io/konflux-project-controller?